src/TagsInput/components/InputField.js

```diff
@@ -4,71 +4,90 @@ import * as React from 'react';
 import { View, TextInput, Text } from 'react-native';
 import type { ViewLayoutEvent } from 'react-native/Libraries/Components/View/ViewPropTypes';
 import { StyleSheet } from '../../PlatformStyleSheet';
+import { INPUT_MIN_WIDTH } from '../constants';
 
 type Props = {|
-  +onChangeText: (value: string) => void,
-  +value: string,
-  +onLayout: (e: ViewLayoutEvent) => void,
-  +maxWidth: ?number | 'auto',
-  +minWidth: number,
   +fontSize: number,
-  +disabled: ?boolean,
-  +placeholder: ?string,
-  +forwardRef: (el: any) => void,
-  +onFocus: () => void,
-  +onBlur: () => void,
-  +autoFocus?: boolean,
+  +onChangeText: (value: string) => void,
+  +forwardRef?: (el: any) => void,
+  +minWidth?: number,
+  +value?: string,
+  +onFocus?: () => void,
+  +onBlur?: () => void,
+  +disabled?: boolean,
+  +maxWidth?: number,
+  +placeholder?: string,
+  +onKeyPress?: (e: ViewLayoutEvent) => void,
 |};
 
 type State = {|
-  componentWidth: number | 'auto',
+  componentWidth: number,
 |};
 
-class InputField extends React.PureComponent<Props, State> {
+class InputField extends React.Component<Props, State> {
   static defaultProps = {
-    maxWidth: 'auto',
-    minWidth: 'auto',
+    fontSize: 16,
```

Comment:
I think we could use here `fontSizeTextLarge` from the design tokens.

Reply:
done